_studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp

```diff
@@ -833,11 +833,14 @@ namespace
 #if defined(MFX_ENABLE_MFE)
     mfxU16 GetDefaultNumMfeFrames(mfxU32 targetUsage, const mfxFrameInfo& info,
         eMFXHWType platform, mfxFeiFunction func, int slices, bool extSurfUsed,
-        eMFXGTConfig config)
+        eMFXGTConfig config, mfxU16 brcMethod)
     {
         targetUsage;//no specific check for TU now, can be added later
         if (platform <= MFX_HW_BDW)//no MFE support prior to SKL.
             return 1;
+        // ICQ is unsupported for MFE now, QVBR and VCM are low latency usage rate controls, MFE is not suitable for low latency usages
```

Comment:
I believe a warning log message is better than as silent comment

Reply:
Warning will be returned at encoder initialization

Reply:
Does anyone know something is really happening?  If you checked https://github.com/Intel-Media-SDK/MediaSDK/issues/863, there is already a warning when MFE is disabled but nobody know what is the precise warning is.  Please make things easily to be checked/debugged, no everyone user is built from source code. Log message is fat better than "Hi, code is here, please debug it from source code"